libraries/RC_Channel/examples/RC_Channel/RC_Channel.cpp

```diff
@@ -18,68 +10,54 @@ const AP_HAL::HAL& hal = AP_HAL::get_HAL();
 
 #define NUM_CHANNELS 8
 
-static RC_Channel rc_1(CH_1);
-static RC_Channel rc_2(CH_2);
-static RC_Channel rc_3(CH_3);
-static RC_Channel rc_4(CH_4);
-static RC_Channel rc_5(CH_5);
-static RC_Channel rc_6(CH_6);
-static RC_Channel rc_7(CH_7);
-static RC_Channel rc_8(CH_8);
-static RC_Channel **rc = RC_Channel::rc_channel_array();
+static RC_Channels rc_channels;
+static RC_Channel *rc;
 
 static void print_pwm(void);
 static void print_radio_values();
-static void copy_input_output(void);
 
 
 void setup()
 {
     hal.console->println("ArduPilot RC Channel test");
 
+    rc = RC_Channels::rc_channel(CH_1);
```

Comment:
rc is initialised with a reference to CH_1, but in the lines below this is treated as an array and indexed with CH_2 , CH_3 etc.    seems weird . should this variable be pluralised like 'rc_channels' is ?